### PR TITLE
added deploy command

### PR DESCRIPTION
**Pull request by nrdurkin**

### CODE REVIEW

**Comment by bbarber9** [comment]
@nrdurkin Do we want to merge this into issue96 or into dev?

**Comment by nrdurkin** [comment]
I'm not super familiar with how git branches should be setup, but these changes use and slightly modify the functions I wrote for issue96.

**Comment by bbarber9** [comment]
I think in this case we will want to merge in to dev, but merge it after your build PR goes through. That way we can check each piece individually

**Comment by collinparker-nuvolo** [comment]
@nrdurkin - there are a few files in this PR that have merge conflicts with the dev branch that need to be resolved. The best way to do this is switch to this branch locally in vscode and run `git merge origin dev`, then clean up the conflicts and save each file, then push up the merge commit. Let me know if you need help with this.

File: packages/core/src/AppManager.ts [hunk 1]
```diff
@@ -305,40 +306,14 @@ class AppManager {
...
-      }, []);
-      logger.silly(`${paths.length} paths found...`);
-      logger.silly(JSON.stringify(paths, null, 2));
-      let fileContexts = await this.parseFileParams(paths);
-      logger.info(`${fileContexts.length} files to push...`);
-      logger.silly(
-        JSON.stringify(fileContexts.map(ctx => ctx.filePath), null, 2)
-      );
-      if (skipPrompt || (await this.canPush())) {
+    if (skipPrompt || (await this.canPush())) {
```
[the remark this answers is not in the document]
Comment (nrdurkin) [reply packages/core/src/AppManager.ts:309]: Yes it would

**Pull request merged**